docs/pages/reference/agent-services/agent_services_glossary.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Agent Services and Clients Glossary
+h1: Agent Service Glossary
+description: Glossary for Teleport Agent Services and Clients.
+---
+
+# Teleport Agent services and clients
+
+This glossary provides definitions and explanations for key terms related to Teleport Agent services and clients. Teleport Agent runs on 
+target systems to facilitate secure access through the Teleport cluster. Most agent services can be enabled together or separately depending on deployment needs. 
+The **"Enabled by Default"** column indicates whether each service is active by default when you run `teleport start` without specifying services.
+
+## Teleport Agent services
+
+| Service Name  | Description   | Common Use Case  | Enabled by Default |
+| ----------------- | ------------------------------------------------------------------------------ | -------------------------------------------------- | ------------------ |
+| `auth_service` | Core authentication and certificate authority service. | Central identity and policy engine  | ✅ |
+| `proxy_service` | Routes client requests and handles access to Teleport clusters and resources. | Public access point for users and workloads | ✅ |
+| `ssh_service` | Provides SSH access to Linux servers with auditing and session recording. | Secure access to Linux nodes | ✅ |
+| `kubernetes_service` | Provides secure, audited access to Kubernetes clusters. | Access control and session tracking for Kubernetes | ❌ |
+| `app_service` | Provides secure access to internal web applications.                           | Protect internal tools and dashboards | ❌ |
+| `db_service` | Manages identity-aware access to databases (e.g., PostgreSQL, MySQL, MongoDB). | Role-based database access with auditing | ❌ |
+| `discovery_service` | Automatically discovers and joins cloud resources (e.g., EC2, Azure VMs). | Auto-enroll infrastructure | ❌ |
+| `windows_desktop_service` | Enables RDP access to Windows desktops through Teleport. | Access control and recording for Windows systems | ❌ |
+| `jamf_service` | Integrates with Jamf for device trust enforcement. | Verify macOS device compliance before login | ❌ |
+| `debug_service` | Exposes internal metrics and debug endpoints for diagnostics. | Development, profiling, and debugging | ✅ |
+
+---
+
+### Auth Service
+
+These settings apply to the Teleport Auth Service:
+
+<Admonition type="tip">
+Teleport Enterprise Cloud manages the Auth Service for you, so you do not need
+to specify these configuration settings.
+</Admonition>
+
+```yaml
+(!docs/pages/includes/config-reference/auth-service.yaml!)
```

Comment:
Maybe its just me but I would expect glossary page to just explain all the product specific terminology and have the configuration reference in a separate reference page.